views/view-content.php

```diff
@@ -0,0 +1,171 @@
+	</nav>
+
+	<h2></h2>
+
+	<form id="statify-dashboard-controls">
+		<input type="hidden" name="page" value="statify_content">
+		<?php
+		if ( 'popular' !== $selected_type ) {
+			echo '<input type="hidden" name="type" value="' . esc_attr( $selected_type ) . '">';
+		}
+		?>
+		<label for="statify-content-daterange"><?php esc_html_e( 'Date range', 'statify' ); ?></label>
+		<select id="statify-content-daterange" name="range">
+			<option value=""><?php esc_html_e( 'default (all the time)', 'statify' ); ?></option>
+			<option value="lastYear" <?php selected( $date_range, 'lastYear' ); ?>><?php esc_html_e( 'last year', 'statify' ); ?></option>
+			<option value="lastWeek" <?php selected( $date_range, 'lastWeek' ); ?>><?php esc_html_e( 'last week', 'statify' ); ?></option>
+			<option value="yesterday" <?php selected( $date_range, 'yesterday' ); ?>><?php esc_html_e( 'yesterday', 'statify' ); ?></option>
+			<option value="today" <?php selected( $date_range, 'today' ); ?>><?php esc_html_e( 'today', 'statify' ); ?></option>
+			<option value="thisWeek" <?php selected( $date_range, 'thisWeek' ); ?>><?php esc_html_e( 'this week', 'statify' ); ?></option>
+			<option value="last28days" <?php selected( $date_range, 'last28days' ); ?>><?php esc_html_e( 'last 28 days', 'statify' ); ?></option>
+			<option value="lastMonth" <?php selected( $date_range, 'lastMonth' ); ?>><?php esc_html_e( 'last month', 'statify' ); ?></option>
+			<option value="thisMonth" <?php selected( $date_range, 'thisMonth' ); ?>><?php esc_html_e( 'this month', 'statify' ); ?></option>
+			<option value="thisYear" <?php selected( $date_range, 'thisYear' ); ?>><?php esc_html_e( 'this year', 'statify' ); ?></option>
+			<option value="1stQuarter" <?php selected( $date_range, '1stQuarter' ); ?>><?php esc_html_e( '1st quarter', 'statify' ); ?></option>
+			<option value="2ndQuarter" <?php selected( $date_range, '2ndQuarter' ); ?>><?php esc_html_e( '2nd quarter', 'statify' ); ?></option>
+			<option value="3rdQuarter" <?php selected( $date_range, '3rdQuarter' ); ?>><?php esc_html_e( '3rd quarter', 'statify' ); ?></option>
+			<option value="4thQuarter" <?php selected( $date_range, '4thQuarter' ); ?>><?php esc_html_e( '4th quarter', 'statify' ); ?></option>
+			<option value="custom" <?php selected( $date_range, 'custom' ); ?>><?php esc_html_e( 'custom', 'statify' ); ?></option>
+		</select>
+		<label for="statify-content-datestart">Start date</label>
+		<input id="statify-content-datestart" name="start" type="date" value="<?php echo esc_attr( $date_start ); ?>">
+		<label for="statify-content-dateend">End date</label>
+		<input id="statify-content-dateend" name="end" type="date" value="<?php echo esc_attr( $date_end ); ?>">
+		<button type="submit" class="button-secondary"><?php esc_html_e( 'Select date period', 'statify' ); ?></button>
+	</form>
+
+	<?php if ( 'popular' === $selected_type ) : ?>
+
+	<section>
+		<h3><?php esc_html_e( 'Most Popular Content', 'statify' ); ?></h3>
```

Comment:
Because there’s no `h2` before, this needs to be one

```suggestion
		<h2><?php esc_html_e( 'Most Popular Content', 'statify' ); ?></h2>
```